src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -247,4 +249,23 @@ public void repoConfig_defaultBranch_success() throws ParseException, IOExceptio
         Assert.assertEquals(expectedConfig.getBranch(), actualConfigs.get(0).getBranch());
         Assert.assertEquals(expectedConfig.getBranch(), authorConfigs.get(0).getBranch());
     }
+
+    @Test
+    public void repoConfig_overrideKeyword_success() throws ParseException, IOException, HelpScreenException {
+        RepoConfigCsvParser repoConfigCsvParser = new RepoConfigCsvParser(REPO_CONFIG_OVERRIDE_KEYWORD_FILE);
+        List<RepoConfiguration> configs = repoConfigCsvParser.parse();
+        RepoConfiguration config = configs.get(0);
+
+        Assert.assertEquals(1, configs.size());
+        Assert.assertEquals(new RepoLocation(TEST_REPO_BETA_LOCATION), config.getLocation());
+        Assert.assertEquals(TEST_REPO_BETA_BRANCH, config.getBranch());
+        Assert.assertEquals(TEST_REPO_BETA_CONFIG_FORMATS, TEST_REPO_BETA_CONFIG_FORMATS);
+        Assert.assertFalse(config.isStandaloneConfigIgnored());
+        Assert.assertEquals(config.getIgnoreCommitList(),
+                CommitHash.convertStringsToCommits(TEST_REPO_BETA_CONFIG_IGNORED_COMMITS));
```

Comment:
By convention, when asserting 2 elements are equal, 'expected' should be the 1st argument, and the actual should be the 2nd.